docs/reference/ccr/overview.asciidoc

```diff
@@ -102,9 +141,71 @@ Using these APIs in tandem enables you to adjust the read and write parameters
 on the follower shard task if your initial configuration is not suitable for
 your use case.
 
+[float]
+=== Leader index retaining operations for replication
+
+If the follower is unable to replicate operations from a leader for a period of
+time, the following process can fail due to the leader lacking a complete history
+of operations necessary for replication.
+
+Operations replicated to the follower are identified using a sequence number
+generated when the operation was initially performed. Lucene segment files are
+occasionally merged in order to optimize searches and save space. When these
+merges occur, it is possible for operations associated with deleted or updated
+documents to be pruned during the merge. When the follower requests the sequence
+number for a pruned operation, the process will fail due to the operation missing
+on the leader.
+
+This scenario is not possible in an append-only workflow. As documents are never
+deleted or updated, the underlying operation will not be pruned.
+
+Elasticsearch attempts to mitigate this potential issue for update workflows using
+a Lucene feature called soft deletes. When a document is updated or deleted, the
+underlying operation is retained in the Lucene index for a period of time. This
+period of time is governed by the `index.soft_deletes.retention_lease.period`
+setting which defaults to `12 hours`.
+
+When a follower initiates the index following, it acquires a retention lease from
+the leader. This informs the leader that it should not allow a soft delete to be
+pruned until either the follower indicates that it has received the operation or
+the lease expires after `12 hours`. It is valuable to have monitoring in place to
```

Comment:
I find the formatting on "12 hours" odd. Is there a reason that you chose to format it this way, as opposed to "twelve hours" (without being formatted as code) or `12h`. Another concern I have is if we change the defaults from `12h` to some other value and we would have to maintain all the pages in the docs. Maybe the best option is to add docs for the `index.soft_deletes.retention_lease.period` setting (or link to the CCR requirements page) and let the default be only specified there?

Reply:
I linked the requirements page.